### PR TITLE
updating the composer files

### PR DESCRIPTION
**Pull request by levidehaan**
learning drupal7-framework with Doug and found we could do an update so we could go over the git-flow workflow.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,10 +4,6 @@
     "repositories": [
         {
             "type": "composer",
-            "url": "http://static.drupal-packagist.org/v0.2.1"
-        },
-        {
-            "type": "composer",
```
Comment (dsdobrzynski) [inline composer.json]: We'll want to keep both repositories.

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,5 @@ vendor
 .bundle
 .ruby-version
 .sass-cache
+.iml
+.idea
```
Comment (dsdobrzynski) [inline .gitignore]: In the future, I would make the changes to gitignore and updates two separate commits.

File: composer.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
             "type": "composer",
             "url": "http://static.drupal-packagist.org/v0.2.1"
         },
-        {
+	{
```
Comment (dsdobrzynski) [inline composer.json]: One last small item: Looks like you lost a space here.

**Comment by levidehaan** [comment]
This pull request doesn't actually do anything, removing.

**Pull request closed without merging**